_std/defines/projectiles.dm

```diff
@@ -4,7 +4,7 @@
 #define PROJ_PASSOBJ			(1<<2)
 
 //pass flags on hit thing take priotity if they exist
-#define PROJ_ATOM_PASSTHROGH	(1<<4)
+#define PROJ_ATOM_PASSTHROUGH	(1<<4)
```

Comment:
thanks for fixing this typo. my standard library review blessing is now upon you

Reply:
np =3